### ISSUE TITLE
Text width makes certain docs unusable

### ISSUE THREAD

**Issue by ajackson-cpi** [golang/gddo#458]
Reading this page is unusable as one must scroll to the end to navigate left-right per line read:
https://godoc.org/github.com/nicksnyder/go-i18n/goi18n

**Comment by shurcooL** [comment]
This looks like a potential issue with `github.com/nicksnyder/go-i18n/goi18n` package, not godoc.org. godoc.org is correctly displaying documentation for that package the way the `godoc` command would, isn't it?

If you, you should report it in its [issue tracker](https://github.com/nicksnyder/go-i18n/issues).

**Comment by ajackson-cpi** [comment]
Yes, so should I make this into a GoDoc bug? This is unreadable. Some sort of wrapping or expansion should occur in this case.

**Comment by shurcooL** [comment]
You can try, but as far as I can tell, this is working as expected. It makes sense for pre-formatted text to appear without wrapping, just horizontal scrolling if needed. E.g.:

[![image](https://cloud.githubusercontent.com/assets/1924134/20604694/b75cf968-b235-11e6-9ed6-7fbe3ef9c3ee.png)](http://instantshare.virtivia.com:27080/4r28fpq4btj0.mov)
